lib/pagem.rb

```diff
@@ -14,7 +13,11 @@ def initialize(scope, params, opt = {})
     @params = params
     @items_per_page = opt[:items_per_page] || ITEMS_PER_PAGE
   end
-  
+
+  def icon_link(icon, text, url, options = {})
+    link_to text, url, options.merge(class: "#{options[:class] if options[:class]} iconlink", style: "background-image:url(" + (options[:globalurl] ? icon.to_s : "/images/medidata_buttons/#{icon}.gif") + ")", globalurl: nil)
```

Comment:
You could leave out the ```if options[:class]``` and break this into multiple lines for readability and to be within 120 characters per our [Ruby style guide](https://github.com/mdsol/ruby-style-guide):

```ruby
def icon_link(icon, text, url, options={})
  options = options.merge(
    class: "#{options[:class]} iconlink",
    style: %[background-image:url(#{options[:global_url] ? icon.to_s : "/images/medidata_buttons/#{icon}.gif"})],
    global_url: nil)
  link_to(text, url, options)
end
```

Also, minor point, but ```global_url``` would follow Ruby naming conventions more.